content/api/commands/intercept.md

```diff
@@ -2,89 +2,84 @@
 title: intercept
 ---
 
-Cypress routes all HTTP requests - including `XMLHttpRequest` (XHR) and `fetch` - through its proxy. Use `cy.intercept` to manage the behavior of these requests, including:
+Capture or change the behavior or contents of network requests and responses.
 
-<!-- TODO DX-182 add links to examples -->
+<Alert type="warning">
+
+We highly recommend you read the [Network Requests](/guides/guides/network-requests) guide first.
 
-- Waiting on HTTP requests to complete before executing commands.
-- Making assertions and modifying (statically or dynamically):
-  - the request made by your front-end application.
-  - the response from your back-end service.
-- Mocking all or some of your backend API by stubbing out responses.
-- Simulating different client connections by throttling data transfer rate.
-- Simulating back-end service bottlenecks by adding a delay to the response.
-- Simulating a 3rd-party API outage by forcing a network error.
+</Alert>
```

Comment:
I think a better approach to point 2 would be to have a callout at the top of the page, like on the [stub](https://docs.cypress.io/api/commands/stub) and [spy](https://docs.cypress.io/api/commands/spy) pages that links to the [Network Requests](https://docs.cypress.io/guides/guides/network-requests) guide.

Reply:
> In the alert box titles, we can say things like "Note", "Tip", or "Consider X." Or some title that's contextually relevant to the info in the box.

Good catch. I'll update those callouts and update [Doc Standards](https://www.notion.so/cypressdx/Documentation-Standards-WIP-f9ed5c4e5bdb44aabc4b13814bd693fa)

> * [ ]   Move GraphQL examples into the main examples section. They don't need their own top-level placement.

I can move this to the main examples section, but this is already a huge doc and there's a separate guide for this content. I'll need to trim it down to a single example with a brief comment and a link to the dedicated guide.

> @cowboy and @mccataldo It's not clear to me what the confusion would be with the types and documentation of `StaticResponse`. Is the concern that it can be both a `string` and an `object`? Can you please elaborate?

We clarified this offline. 

> 1. Let's use the "stub" and "spy" terminology as we do everywhere else.

Sounds good. I'll update it. 

> 2. We have to put back the use cases in the intro. I know this is not consistent with all the other commands, but we have to thoughtfully make exceptions where it makes sense.

I agree with @cowboy on this one. However, I want to point out that the Network Requests guide doesn't contain those Intercept use cases so when I removed that content I created [DX-320](https://linear.app/cypress/issue/DX-320/update-guide-network-requests) to address that.

Reply:
Agree with both of your comments above. So we're good to move forward after all these updates.

As a general rule of thumb, we should interlink to other docs whenever appropriate, which is most of the time.